diff -Naur bart-0.6.00.orig/Makefile bart-0.6.00/Makefile
--- bart-0.6.00.orig/Makefile	2020-07-30 15:31:14.030998455 -0400
+++ bart-0.6.00/Makefile	2020-07-30 15:32:59.422001732 -0400
@@ -315,7 +315,7 @@
 
 # sm_20 no longer supported in CUDA 9
 GPUARCH_FLAGS ?= 
-NVCCFLAGS = -DUSE_CUDA -Xcompiler -fPIC -Xcompiler -fopenmp -O3 $(GPUARCH_FLAGS) -I$(srcdir)/ -m64 -ccbin $(CC)
+NVCCFLAGS = -DUSE_CUDA -Xcompiler -fPIC -Xcompiler -fopenmp -O3 $(GPUARCH_FLAGS) -I$(srcdir)/ -m64 $(NVCCFLAGS_LOCAL)
 #NVCCFLAGS = -Xcompiler -fPIC -Xcompiler -fopenmp -O3  -I$(srcdir)/
 
 
diff -Naur bart-0.6.00.orig/Makefiles/Makefile.cuda bart-0.6.00/Makefiles/Makefile.cuda
--- bart-0.6.00.orig/Makefiles/Makefile.cuda	1969-12-31 19:00:00.000000000 -0500
+++ bart-0.6.00/Makefiles/Makefile.cuda	2020-07-30 15:31:58.149999827 -0400
@@ -0,0 +1,3 @@
+CUDA=1
+CUDA_BASE=/opt/cuda/
+NVCCFLAGS_LOCAL=@@GENTOO_NVCC_FLAGS@@
