diff -Naur bart-0.6.00.orig/src/nufft.c bart-0.6.00/src/nufft.c
--- bart-0.6.00.orig/src/nufft.c	2021-02-03 21:39:53.195666796 -0500
+++ bart-0.6.00/src/nufft.c	2021-02-03 21:41:00.269661581 -0500
@@ -55,6 +55,10 @@
 
 	float lambda = 0.;
 
+    // input options
+    bool scale_im = false;
+    float scaling = 0.;
+
 	const struct opt_s opts[] = {
 
 		OPT_SET('a', &adjoint, "adjoint"),
@@ -69,6 +73,9 @@
 		OPT_SET('P', &conf.periodic, "periodic k-space"),
 		OPT_SET('s', &dft, "DFT"),
 		OPT_SET('g', &gpu, "GPU (only inverse)"),
+        OPT_INT('v', &debug_level, "level", "Verbose level (debug)"),
+        OPT_FLOAT('w', &scaling, "val", "inverse scaling of the data"),
+        OPT_SET('S', &scale_im, "re-scale the image after reconstruction"),
 		OPT_CLEAR('1', &conf.decomp, "use/return oversampled grid"),
 	};
 
@@ -143,6 +150,12 @@
 			linop_adjoint(nufft_op, DIMS, coilim_dims, img, DIMS, ksp_dims, ksp);
 		}
 
+        // try scaling
+        if (scale_im) {
+            debug_printf(DP_DEBUG1, "Re-scaling...");
+            md_zsmul(DIMS, coilim_dims, img, img, scaling);
+        }
+
 		linop_free(nufft_op);
 		unmap_cfl(DIMS, ksp_dims, ksp);
 		unmap_cfl(DIMS, coilim_dims, img);
